tasks/deploy.js

```diff
@@ -0,0 +1,194 @@
+    daoName: process.env.DAO_NAME,
+    owner: accounts[0],
+  });
+}
+
+async function deployTestNetworkDao(deployFunction, network, hardhatImports) {
+  if (!process.env.DAO_NAME)
+    throw Error("Missing env var: DAO_NAME");
+  if (!process.env.DAO_OWNER_ADDR)
+    throw Error("Missing env var: DAO_OWNER_ADDR");
+  if (!process.env.ERC20_TOKEN_NAME)
+    throw Error("Missing env var: ERC20_TOKEN_NAME");
+  if (!process.env.ERC20_TOKEN_SYMBOL)
+    throw Error("Missing env var: ERC20_TOKEN_SYMBOL");
+  if (!process.env.ERC20_TOKEN_DECIMALS)
+    throw Error("Missing env var: ERC20_TOKEN_DECIMALS");
+  if (!process.env.COUPON_CREATOR_ADDR)
+    throw Error("Missing env var: COUPON_CREATOR_ADDR");
+
+  return await deployDao({
+    ...hardhatImports,
+    deployFunction,
+    unitPrice: toBN(toWei("100", "finney")).toString(),
+    nbUnits: toBN("100000").toString(),
+    tokenAddr: ETH_TOKEN,
+    erc20TokenName: process.env.ERC20_TOKEN_NAME,
+    erc20TokenSymbol: process.env.ERC20_TOKEN_SYMBOL,
+    erc20TokenDecimals: process.env.ERC20_TOKEN_DECIMALS,
+    maxChunks: toBN("100000").toString(),
+    votingPeriod: 600, // 600 secs = 10 mins
+    gracePeriod: 600, // 600 secs = 10 mins
+    offchainVoting: true,
+    chainId: getNetworkDetails(network).chainId,
+    deployTestTokens: true,
+    finalize: false,
+    maxExternalTokens: 100,
+    couponCreatorAddress: process.env.COUPON_CREATOR_ADDR,
+    daoName: process.env.DAO_NAME,
+    owner: process.env.DAO_OWNER_ADDR,
+    offchainAdmin: "0xedC10CFA90A135C41538325DD57FDB4c7b88faf7",
+  });
+}
+
+async function deployGanacheDao(deployFunction, network, accounts, hardhatImports) {
+  if (!process.env.DAO_NAME)
+    throw Error("Missing env var: DAO_NAME");
+  if (!process.env.COUPON_CREATOR_ADDR)
+    throw Error("Missing env var: COUPON_CREATOR_ADDR");
+  if (!process.env.ERC20_TOKEN_NAME)
+    throw Error("Missing env var: ERC20_TOKEN_NAME");
+  if (!process.env.ERC20_TOKEN_SYMBOL)
+    throw Error("Missing env var: ERC20_TOKEN_SYMBOL");
+  if (!process.env.ERC20_TOKEN_DECIMALS)
+    throw Error("Missing env var: ERC20_TOKEN_DECIMALS");
+
+  return await deployDao({
+    ...hardhatImports,
+    deployFunction,
+    unitPrice: toBN(toWei("100", "finney")).toString(),
+    nbUnits: toBN("100000").toString(),
+    tokenAddr: ETH_TOKEN,
+    erc20TokenName: process.env.ERC20_TOKEN_NAME,
+    erc20TokenSymbol: process.env.ERC20_TOKEN_SYMBOL,
+    erc20TokenDecimals: process.env.ERC20_TOKEN_DECIMALS,
+    maxChunks: toBN("100000").toString(),
+    votingPeriod: 120, // 120 secs = 2 mins
+    gracePeriod: 60, // 60 secs = 1 min
+    offchainVoting: true,
+    chainId: getNetworkDetails(network).chainId,
+    deployTestTokens: true,
+    finalize: false,
+    maxExternalTokens: 100,
+    couponCreatorAddress: process.env.COUPON_CREATOR_ADDR,
+    daoName: process.env.DAO_NAME,
+    owner: '0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openlawteam/tribute-contracts/pullRequest?prid=8819147)
```suggestion
    owner: "0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1",
```